### PR TITLE
Backport: NetBSD build stuffs

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

**Comment by UdjinM6** [comment]
Needs at least some trivial dashification. Also, pretty sure these instructions won't work for Dash - you need to build with `depends` or build/install `bls_dash` and `backtrace` or it won't compile otherwise.

File: doc/build-netbsd.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+NetBSD build guide
+======================
+(updated for NetBSD 8.0)
+
+This guide describes how to build dashd and command-line utilities on NetBSD.
+
+This guide does not contain instructions for building the GUI.
+
+**This guide has not been tested for building Dash Core and is expected to fail due to missing `bls_dash` and `backtrace`. Please report your results; contributions welcome.**
```
Comment (UdjinM6) [inline doc/build-netbsd.md:9]: It's a bit awkward to include instructions that do not work... Why do we want these PRs to be backported in the first place if instructions are useless?

File: doc/build-unix.md [hunk 2]
```diff
@@ -2,7 +2,8 @@ UNIX BUILD NOTES
 ====================
 Some notes on how to build Dash Core in Unix.
 
-(for OpenBSD specific instructions, see [build-openbsd.md](build-openbsd.md))
+(For BSD specific instructions, see [build-openbsd.md](build-openbsd.md) and/or
```
Comment (UdjinM6) [inline doc/build-unix.md:5]: Side note: we do not have `build-openbsd.md`, our docs mentioning it are misleading. We have a couple of sections for OpenBSD in `build-unix.md` instead.

File: doc/build-netbsd.md (hunk 1 above)
Comment (PastaPastaPasta) [reply doc/build-netbsd.md:9]: Well, I don't think they're useless, they're just not complete and noone here seems invested in completing them. I think there isn't much harm in backporting them

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**